configs/agents/deep_research_agent.yaml

```diff
@@ -0,0 +1,234 @@
+        
+        **Citation Requirements:**
+        - The deep_research tool handles citations automatically
+        - Citations are embedded in the generated report
+        - Each source is tracked with relevance scores
+        - Users can view citation details in the frontend
+        
+        **When to Use Deep Research:**
+        - Complex topics requiring comprehensive coverage
+        - Questions needing multiple perspectives
+        - Research requiring both web and internal sources
+        - Topics where quality and depth matter
+        
+        **Research Process:**
+        The tool will show real-time progress through:
+        - Planning: Generating initial queries
+        - Searching: Finding sources across multiple platforms
+        - Reflecting: Assessing quality and identifying gaps
+        - Synthesizing: Creating the final report
+        
+        Remember: Always check for and use the user's deep_research_settings from metadata!
+      
+      tools:
+        - tool_type: builtin
+          tool_name: deep_research
+          tool_config:
+            model_configs:
+              report_generation:
+                model: "openai/gpt-5-nano"
+                temperature: 1
+                timeout: 300
+
+
+      session_service: *default_session_service
+      artifact_service: *default_artifact_service
+      
+      artifact_handling_mode: "embed"
+      enable_embed_resolution: true
+      enable_artifact_content_instruction: true
+      enable_auto_continuation: true
+      max_llm_calls_per_task: 30  # Higher limit for iterative research
+      data_tools_config: *default_data_tools_config
+
+      # Agent Card Definition
+      agent_card:
+        version: "1.0.0"
+        description: |
+          Advanced research agent that conducts comprehensive, iterative research across web and knowledge base sources.
+          
+          Capabilities:
+          • Deep Research: Iterative research with LLM-powered reflection and query refinement
+          • Web Search: Google and Tavily search integration for public information
+          • Knowledge Base Search: Internal document search capabilities
+          • Citation Tracking: Automatic source tracking with relevance scores
+          • Quality Assessment: LLM evaluates research completeness and identifies gaps
+          • Report Generation: Comprehensive reports with proper citations
+          
+          Perfect for:
+          - Complex research topics requiring depth and breadth from public sources
+          - Multi-perspective analysis using web and internal documents
+          - Comprehensive information gathering from web and knowledge bases
+          - Research requiring both public web sources and internal documentation
+          - Topics where citation and source quality matter
+          
+          Note: This is the web-only version. Requires Google or Tavily API keys for web search.
```

Comment:
We should think a bit about this agent card. While it for sure has utility for people to read, this description is going to be put into the context of any agents that are able to call it. We need the right balance of understanding what it can do (probably best suited for skills) but not too much text that is not useful to the calling agent (e.g. orchestrator). We can put more text in the documentation for deep research for people to read

Definitely, saying you need keys is not useful in this context